.travis.yml

```diff
@@ -28,7 +28,7 @@ matrix:
     - python: pypy
       env: TOXENV=pypy
     - env: TOXENV=py27-flake8
-    - env: TOXENV=py34-flake8
+    - env: TOXENV=py36-flake8
```

Comment:
This may need to look like:

```yaml
- env: TOXENV=py36-flake8
  python: 3.6
```

Based on the Travis failures

Reply:
I changed the `py27-flake8` as well to explicitly use Python 2.7. I've also moved them under their respective Python versions.